sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -466,7 +467,7 @@ async def connect(self):
 
             self.sock = await self.session.ws_connect(
                 url=url,
-                timeout=self.socket_timeout,  # timeout for connect
+                timeout=ClientWSTimeout(ws_close=self.socket_timeout),
```

Comment:
could this raise any other unaccounted-for errors?

Reply:
Its following what aiohttp does when you pass in a float, so the behavior should be the same https://github.com/aio-libs/aiohttp/blob/dbbcc619fa2db09bb800f81a2f2a94277d84fb11/aiohttp/client.py#L891-L898